packages/0xcert-ethereum-erc721-contracts/src/contracts/nf-token-metadata-enumerable.sol

```diff
@@ -646,7 +602,7 @@ contract NFTokenMetadataEnumerable is
     j = _i;
     while (j != 0)
     {
-      bstr[k--] = byte(uint8(48 + j % 10));
+      bstr[k--] = bytes1(uint8(48 + j % 10));
```

Comment:
@fulldecent I need some help with this function. It does not work with 0.8.0.

Reply:
@MoMannn  Fixed at https://github.com/MoMannn/framework-1/pull/3

Reply:
Yep that works :D